src/main/java/net/coderbot/iris/gl/image/ImageLimits.java

```diff
@@ -10,7 +11,7 @@
 
 	private ImageLimits() {
 		if (GL.getCapabilities().OpenGL42) {
-			this.maxImageUnits = GL42C.glGetInteger(GL42C.GL_MAX_IMAGE_UNITS);
+			this.maxImageUnits = GlStateManager._getInteger(GL42C.GL_MAX_IMAGE_UNITS);
 		} else if (GL.getCapabilities().GL_EXT_shader_image_load_store) {
 			this.maxImageUnits = EXTShaderImageLoadStore.GL_MAX_IMAGE_UNITS_EXT;
```

Comment:
Just noticed this but this was a bug with the image load store PR, it should use getInteger too.